cookbooks/bcpc-hadoop/templates/default/zabbix.zbx_mail.sh.erb

```diff
@@ -4,5 +4,6 @@ cluster_name=$2
 severity=$3
 description=$4
 host=$5
+owner=$6
 
-logger -t "Zabbix-Trigger-Action" <<< "Please attend to severity $severity issue with $trigger_name on $cluster_name:$host; problem description=$description"
+logger -t "Zabbix-Trigger-Action" <<< "@$owner: Please attend to severity $severity issue with $trigger_name on $cluster_name:$host; problem description=$description"
```

Comment:
I was going to write, I don't understand `@$owner` but typing `@` in GitHub, I think I understand this is a reference to a person in the Twitter-style?